### DISCUSSION TITLE
Tailwind isn't picking up changes from a directory called "test-results"

### DISCUSSION THREAD

**Discussion by SjorsO** [tailwindlabs/tailwindcss#13534] [Help]
I'm using Tailwind 3.4.3 with Laravel 10 and Vite 4.5.3. I have a blade file in a directory called "test-results" and Tailwind isn't picking up changes from that file. 

When starting Vite the initial build that Tailwind makes does include classes from the blade file in the "test-results" directory. If I edit the blade file while Vite is watching Tailwind doesn't detect the changes and doesn't update the output css file.

Renaming the "test-results" directory solves the issue. The names "test-result", "temp-results", "test" all work, it seems to just be "test-results" that isn't working.

Other details:
- The file: `resources/views/user/test-results/show.blade.php`
- The content pattern in my Tailwind config: `resources/**/*.{html,php,js,jsx,ts,tsx,vue}`

Tailwind and Vite work correctly for the rest of my project, it seems to only not be working for files in a directory called "test-results". Is this a bug?

**Comment by wongjn** [comment]
Could you provide a project that reproduces the unexpected behavior please?
